benchmark_runner/common/clouds/IBM/ibm_operations.py

```diff
@@ -209,8 +209,18 @@ def run_ibm_ocp_ipi_installer(self):
         :return: True if installation success and raise exception if installation failed
         """
         logger.info(f'Starting OCP IPI installer, Start time: {datetime.now().strftime(datetime_format)}')
-        # Must add -t and -o parameters otherwise remote ssh of ansible will not end
-        self.__ssh.run(cmd=f"ssh -o ServerAliveInterval=30 -o ServerAliveCountMax=5 -o StrictHostKeyChecking=no -t -i {self.__container_private_key_path} {self.__user}@{self.__provision_ip} \"{self.__ibm_login_cmd()};{self.__ibm_ipi_install_ocp_cmd()}\" ")
+        # update ssh config
+        self.__ssh.run(f"echo Host provision >> /{self.__user}/.ssh/config")
+        self.__ssh.run(f"echo -e '\t'HostName {self.__provision_ip} >> /{self.__user}/.ssh/config")
+        self.__ssh.run(f"echo -e '\t'User {self.__user} >> /{self.__user}/.ssh/config")
+        self.__ssh.run(f"echo -e '\t'IdentityFile {self.__container_private_key_path} >> /{self.__user}/.ssh/config")
+        self.__ssh.run(f"echo -e '\t'StrictHostKeyChecking no >> /{self.__user}/.ssh/config")
+        self.__ssh.run(f"echo -e '\t'ServerAliveInterval 30 >> /{self.__user}/.ssh/config")
+        self.__ssh.run(f"echo -e '\t'ServerAliveCountMax 5 >> /{self.__user}/.ssh/config")
+        self.__ssh.run(f"chmod 600 /{self.__user}/.ssh/config")
```

Comment:
This is very important and if you hadn't done it may have caused you problems.  It can be done in a single operation, but this is OK.